tools/docgen/main.go

```diff
@@ -248,6 +249,7 @@ func main() {
 		splunk.Plugin(version, nil),
 		stixview.Plugin(version),
 		nistnvd.Plugin(version, nil),
+		snyk.Plugin(version, nil),
```

Comment:
we really should make this a dynamic discovery (for our plugins) with the ability to extend the list with additions from a flat file (or something) 😄 just a note, not in this pull request